backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -292,6 +279,7 @@ namespace /* anonymous */
         return m_pJniEnv;
     }
 
+#if __ANDROID_API__ >= 23
     bool InitializeLocalClientSideUnwinding(JNIEnv* env) {
```

Comment:
Does the java side correctly gate calls to this function? Should be instead always define it, but have it fail for `API < 23`?